client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -118,6 +122,15 @@ const useMenuItems = ( {
 					enabled: isAdmin && ! isWPCOM && ! isWPForTeamsSite,
 					isSelected: itemLinkMatches( path, scanPath( siteSlug ) ),
 				},
+				{
+					icon: shield,
+					path: '/',
+					link: `${ JETPACK_CLOUD_SCAN_HISTORY_LINK }/${ siteSlug }`,
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)![badge](https://img.shields.io/badge/review-nits-green.svg)

It seems that there is a [paths.ts](https://github.com/Automattic/wp-calypso/blob/878157de9e7d80691632757e66c76ffe39c8a031/client/lib/jetpack/paths.ts#L11-L12) file that defines functions returning JP related paths. I thought we could add something like `scanHistoryPath` to this file, but I understand that [constants.ts](https://github.com/Automattic/wp-calypso/blob/update/show-jp-cloud-scan-history-atomic-sites/client/jetpack-cloud/sections/sidebar-navigation/lib/constants.ts) is maintained, so it's also fine as it is. I was just curious about the differences. 😀